File: WhaleModels/Paddock.cs
using System;

namespace WhaleModels
{
    public class Paddock
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public int Condition { get; set; }
        public int CurrentOccupancy { get; set; }
        public int MaxOccupancy { get; set; }
        public ICollection<Food> Trough { get; set; }
        public ICollection<Whale> Whales { get; set; }
        public int RanchId { get; set; }
        public Ranch Ranch { get; set; }
        public int ToyId { get; set; }
        public Toy Enrichment { get; set; }
    }
}

File: WhaleAngular/src/app/models/paddock.ts
import {Food} from "./food"
import {Ranch} from "./ranch"
import {Toy} from "./toy"
import {Whale} from "./whale"


export interface Paddock {
    id: number,
    name: string,
    condition: number,
    currentOccupancy: number,
    maxOccupancy: number,
    trough: Food[],
    whales: Whale[],
    ranchID: number,
    ranch: Ranch,
    toyId: number,
    enrichment: Toy
}